source/manual/pact-testing.html.md

```diff
@@ -109,3 +109,25 @@ different applications.
 
 When an application calls an API directly, not via GDS API Adapters, this application will be the consumer. For example, Publishing API
 is the consumer in the pact tests between it and Content Store, since it makes [direct calls to Content store](https://github.com/alphagov/publishing-api/blob/main/app/adapters/content_store.rb).
+
+## How GDS API Adapters retrieves tests from the Providers
+
+Each provider defines some details of its tests in the `.github/workflows/pact-verify.yml` file. This workflow is used by both
+the provider and the consumer (gds-api-adapters), and behaves differently based on the value of the pact_artifact input, which can
+be confusing if you're unused to it.
```

Comment:
I started reviewing this and struggled to understand it so had a go at rewriting it with some more detail:

> ## How pacts operates as part of continuous integration (CI)
> 
> Consumers (for example GDS API Adapters) define the tests, and thus the contract, between the consumer and the provider (for example Places Manager API). These tests produce JSON files, known as pact artifcats, and describe how an API is expected to behave. These pact artifcats are then run against the provider codebase to confirm a pact.
> 
> As part of a CI process we need to test the pact whenever a consumer or provider changes. This involves executing [a GitHub action](https://github.com/alphagov/places-manager/blob/main/.github/workflows/pact-verify.yml) defined on the Provider. This action recieves different inputs depending on whether it is executed [by the consumer](https://github.com/alphagov/gds-api-adapters/blob/9f01b67805e915145a3cd7cbc80b6d4a7c9524a8/.github/workflows/ci.yml#L101-L105) or by [the provider](https://github.com/alphagov/places-manager/blob/03b06aa8e853020a1e1734786bab9e2ce2e68839/.github/workflows/ci.yml#L60-L64).  Where the consumer provides pact-artificats files generated as a result of changes to the consumer repository and where the provider downloads previously generated pact-artifacts based on the latest state of the consumer from a service, pact broker.

What do you think? 

I'd also suggest we move this section to be above the guide: "Running Pact tests locally" since it seems key information in proceeding with the further sections